### DISCUSSION TITLE
Anyway to hide the currently selected cleaning mode from the card?

### DISCUSSION THREAD

**Discussion by jeremytodd1** [PiotrMachowski/lovelace-xiaomi-vacuum-map-card#369] [Q&A] [answered]
For reference, I'm talking about the red circled area in these two screenshots:

https://i.imgur.com/8PPaLqP.png
https://i.imgur.com/g9VQ8KF.png

That circled area just changes to the same text as whatever cleaning mode (rooms, zone cleanup, etc.) you currently have selected.

There are a couple of issues with this behavior though:
1. It's redundant information. We can already easily tell which mode is selected as the selected mode gets highlighted dark blue. It's always obvious what mode you have selected.
2.  In addition to being repeated information, it's also confusing in its current form. It just looks like another option in the list. Instead of my list easily being recognized as "Rooms, Zone cleanup", my list looks like "Rooms, Rooms, Zone cleanup".

Here is a before and after mockup I made to show the difference:
https://i.imgur.com/BBoXcwS.png

You get the same information in both the before and after while 1) being easier to quickly understand 2)taking up less real estate space on the card.

No functionality will be lost with this and it's a nicer user experience.

Is this possible to do currently?

**Comment by PiotrMachowski** [comment, marked as answer]
Well, currently this layout is broken due to changes in 2022.03, you can check out how it should look like here: [documentation](https://github.com/PiotrMachowski/lovelace-xiaomi-vacuum-map-card#lovelace-vacuum-map-card). Related issue: #365
